titan/output.py

```diff
@@ -35,9 +35,11 @@ def setup_aggregates(params: ObjMap, reportables, classes: List[str]) -> Dict:
     if classes == []:
         base_stats = {
             "agents": 0,
-            "deaths": 0,
             "deaths_hiv": 0,
         }
+        for exit in params.classes.exit:
+            if exit != "none":
```

Comment:
I don't love the special handling around `none`, I think it's cleaner to still show it and it will always be zero

Reply:
do we want to do the stats on the exit class or the exit type?  I think types would potentially be more useful and that better lines up with how we're tracking the deaths in model

Reply:
we'd have to pass those values in here, but that doesn't seem like a huge change